packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucetContainer.tsx

```diff
@@ -13,36 +15,61 @@ export const CollateralFaucet: FunctionComponent<{ faucetState: VaultState }> =
   const mintAmtChangehandler = (event: React.ChangeEvent<HTMLInputElement>): void => {
     setMintAmt(parseFloat(event.target.value));
   };
-  console.log(faucetState);
   return (
     <Layout>
-      <div className="pt-10 bg-blue-600 sm:pt-16 lg:pt-8 lg:pb-14 lg:overflow-hidden text-blue-200 font-bold">
-        Faucet
-      </div>
-      {faucetState.state === "PENDING" && `Deploying: ${faucetState.hash}`}
-      {faucetState.state === "UNCONNECTED" && <Button onClick={faucetState.activateBrowserWallet}>Connect</Button>}
-      {faucetState.state === "WRONG_NETWORK" && <Button onClick={faucetState.switchNetwork}>Switch Network</Button>}
-      {faucetState.state === "READY" && (
-        <div className="p-4">
-          <h3>MINT (Fake USD) Amount</h3>
-          <input className="shadow-md rounded-lg" type="number" placeholder="1000" onChange={mintAmtChangehandler} />
-          <div>Balance: {bnToNum(faucetState.balance)} FakeUSD</div>
-          <Button onClick={() => faucetState.mintToken(mintAmt)}>Mint Token</Button>
+      <Content>
+        <div className="text-blue-600 text-3xl">Faucet</div>
+        {/*match mt-6 USDT button from Vault Position*/}
+        <div className="mt-6">
+          {faucetState.state === "SUCCESS" && <BasicSuccess title="Transaction successful" />}
+          {faucetState.state === "PENDING" && <LoaderContent>Deploying: {faucetState.hash}</LoaderContent>}
```

Comment:
```suggestion
          {faucetState.state === "PENDING" && <LoaderContent>Waiting for transaction: {faucetState.hash}</LoaderContent>}
```

It's not really deploying